update native SDK versions

cesar-sosa-hol opened the pull request.

Armaxis commented on android/src/main/java/com/mobilepaymentssdkreactnative/Utils.kt:
```diff
@@ -72,11 +73,11 @@ fun ReadableMap.readPaymentParameters(): PaymentParameters {
   statementDescription?.let { builder.statementDescription(it) }
   teamMemberId?.let { builder.teamMemberId(it) }
   tipMoney?.let { builder.tipMoney(it) }
+  idempotencyKey?.let { builder.paymentAttemptId(it) }
```
It will cause a lot of confusion if we map `idempotencyKey` to `paymentAttemptId`. 
Instead, the ReadableMap should have a new field `paymentAttemptId` (only currently used by Android)
And the mapping in this class will become
```kotlin
  idempotencyKey?.let { builder.idempotencyKey(it) }
  paymentAttemptId?.let { builder.paymentAttemptId(it) }
```

cesar-sosa-hol commented on android/src/main/java/com/mobilepaymentssdkreactnative/Utils.kt:
```diff
@@ -381,7 +382,6 @@ fun ReaderChangedEvent.toChangedEventMap(): WritableMap {
   return WritableNativeMap().apply {
     putString("cange", change.toChangeString())
```
Thanks, updated

cesar-sosa-hol commented:
> Same from the flutter PR, We also updated ReaderInfo status. We deprecated ReaderInfo.state. Unsure if we should update in the flutter plugin or not.

I added support for new field status in ReaderInfo

matheuskiser reviewed (approved):
LGTM

Armaxis commented on src/models/objects.ts:
```diff
@@ -82,14 +84,15 @@ export type PaymentParameters = {
   teamMemberId?: String;
   tipMoney?: Money;
   totalMoney?: Money;
+  idempotencyKey: String;
+  paymentAttemptId: String;
```
Both should be nullable, because only one of the two should be set.

cesar-sosa-hol replied:
Done
